SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunnerTests/TupleTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  TupleTests.swift
+//  SwiftRustIntegrationTestRunnerTests
+//
+//  Created by Niwaka on 2023/03/12.
+//
+
+import Foundation
+
+import XCTest
+@testable import SwiftRustIntegrationTestRunner
+
+/// Tests tuples
+final class TupleTest: XCTestCase {
+
+    func testSwiftCallsRustTuples() throws {
```

Comment:
Let's put a `/// Verify that ...` comment on every test so that we can quickly understand exactly what every test is trying to accomplish.

I know that it's in the name, but I that it's easier for a new contributor to read prose than test names.